src/AbstractPathMappingRepository.php

```diff
@@ -278,4 +299,54 @@ protected function createVirtualResource($path = null)
 
         return $resource;
     }
+
+    /**
+     * Transform a relative path into an absolute path.
+     *
+     * @param string $absolutePath
+     *
+     * @return string
+     */
+    protected function makePathRelative($absolutePath)
+    {
+        if (0 === strpos($absolutePath, 'l:')) {
```

Comment:
Because under Windows, `l:/directory` will be understood as a partition where it shouldn't. It avoids problems.